### PR TITLE
parameter list and examples to use correct names

### LINKED ISSUE #16: Parameter list and examples in readme.md seems wrong
'days' and 'bannerCaption' don't produce expected result. instead using 'cookieValidity' and  'caption' will do it.

### PR DESCRIPTION
**Pull request by bedo-abn**
fixes #16

### CODE REVIEW

**Comment by andrerom** [comment]
Seems like you did it the right way, just missed link to the issue ;)

review ping @clash82

File: README.md [hunk 1]
```diff
@@ -89,8 +89,9 @@ The following optional parameters can be set as a second argument in an array fo
 Parameter     | Default value                                  | Description
 ------------- | ---------------------------------------------- | -----------
 cookieName    | privacyCookieAccepted                          | Sets your own status cookie name
-days          | 365                                            | Says how many days privacy banner should be hidden when user accepts policy?
-bannerCaption | Cookies help us create a good experience (...) | Sets your own banner message caption
+cookieValidity          | 365                                            | Says how many days privacy banner should be hidden when user accepts policy?
```
Comment (clash82) [inline README.md]: Good work @bedo-abn. Could you please adjust indents between `|` in this table?
`|` should be on the same position in every line - this makes everything more readable in plain text.

**Comment by clash82** [comment]
Cool, it looks much better now +1 (tested)

**Comment by bedo-abn** [comment]
Thanks for this opportunity to try workflow on something very simple :)

**Comment by damianz5** [comment]
+1

**Comment by clash82** [comment]
Thanks @bedo-abn :)

**Comment by bedo-abn** [comment]
glad i could help

**Comment by SylvainGuittard** [comment]
Thanks @bedo-abn

**Pull request merged**